conf/routes

```diff
@@ -1,19 +1,19 @@
-GET     /                                 controllers.Application.index()
+GET     /geodata/search                           controllers.Application.index()
```

Comment:
The index page should be at just `/geodata`.

See prefix section at http://hbz.github.io/#data-module-specification